Formula/rpm.rb

```diff
@@ -24,14 +22,11 @@ class Rpm < Formula
     sha256 x86_64_linux:   "f2fbe2fec1f2d1212c2122560728929445e499f25df89cf8d623f04b993d5903"
   end
 
-  # We need autotools for the Lua patch below. Remove when the patch is no longer needed.
-  depends_on "autoconf" => :build
-  depends_on "automake" => :build
-  depends_on "libtool" => :build
   depends_on "gettext"
   depends_on "libarchive"
   depends_on "libmagic"
   depends_on "lua"
+  depends_on macos: :ventura
```

Comment:
@SMillerDev, why was this added? In a corporate environment where upgrades are controlled, this will effectively break those individuals since there doesn't seem to be a way to install a specific version.

Reply:
I see there is a dependency on `mkfifoat` for rpm itself. Is there a way around this?